src/status_im/chat/events.cljs

```diff
@@ -202,8 +201,7 @@
                                                 :requests (get chat->message-id->request chat-id))]))
                          (into {}))]
           (-> new-db
-              (assoc-in [:message-data :preview] message-previews)
-              (assoc :handler-data (handler-data/get-all))
```

Comment:
Can you comment on why this is not needed anymore?

Reply:
The whole `:handler-data` in app-db + related realm functionality was added by me and nobody else ever modified it, so I'm quite confident in the change. 
Now we are only reading it during chat initialisation, but never writing back.
It was supposed to be a way how to coordinate asynchronous command handlers, but proved super fragile in distributed system like us, we abandoned it and use a different mechanism to enable async-handlers (special async-handler flag for command + signal from jail to resume execution flow)